dcsazure_AzureSQL_to_AzureSQL_mask_pl/README.md

```diff
@@ -0,0 +1,111 @@
+          * Get the metadata needed for the sink table
+          * If no filter needs to be applied:
+            * Call the `dcsazure_AzureSQL_to_AzureSQL_unfiltered_mask_params_df` data flow to generate masking
+              parameters
+            * Call the `dcsazure_AzureSQL_to_AzureSQL_unfiltered_mask_df` data flow, passing in parameters as
+              generated by the generate masking parameters dataflow
+          * If a filter needs to be applied:
+            * Call the `dcsazure_AzureSQL_to_AzureSQL_filtered_mask_params_df` data flow to generate masking
+              parameters using the filter alias
+            * Call the `dcsazure_AzureSQL_to_AzureSQL_filtered_mask_df` data flow, passing in parameters as
+              generated by the generate masking parameters dataflow and the filter as determined by the output of For
+              Each Table To Mask
+        * If the table cannot be written to (i.e. the Data Factory backend doesn't support this table writer version)
+          * Provide a `400` status error code to indicate the failure of this table (with the appropriate message)
+* Note that there is a deactivated activity `Test Filter Conditions` that exists in order to support importing the
+  filter test utility dataflow, this is making it easier to test writing filter conditions leveraging a dataflow debug
+  session
+
+### Variables
+
+If you have configured your database using the metadata store scripts, these variables will not need editing. If you
+have customized your metadata store, then these variables may need editing.
+
+* `METADATA_SCHEMA` - This is the schema to be used for in the self-hosted AzureSQL database for storing metadata (default `dbo`)
+* `METADATA_RULESET_TABLE` - This is the table to be used for storing the discovered ruleset (default `discovered_ruleset`)
+* `METADATA_SOURCE_TO_SINK_MAPPING_TABLE` - This is the table in the metadata schema that will contain the data
+  mapping, defining where unmasked data lives, and where masked data should go (default `adf_data_mapping`)
+* `METADATA_ADF_TYPE_MAPPING_TABLE` - This is the table that maps from data types in various datasets to the
+  associated datatype required in ADF as needed for the pipeline (default `adf_type_mapping`)
+* `BLOB_STORE_STAGING_STORAGE_PATH` - This is a path that specifies where we should stage data as it moves through the
+  pipeline and should reference a storage container in a storage account (default `staging-container`)
+* `DATASET` - This is the way this data set is referred to in the metadata store (default `ADLS`)
+* `CONDITIONAL_MASKING_RESERVED_CHARACTER` - This is a string (preferably a character) reserved as for shorthand for
+  when referring to the key column when defining filter conditions, in the pipeline this will be expanded out to use the
+  ADF syntax for referencing the key column (default `%`)
+* `TARGET_BATCH_SIZE` - This is the target number of rows per batch (default `2000`)
```

Comment:
This is missing from the pipeline (more on this later).

Reply:
Used this variable as value for `DF_NUMBER_OF_ROWS_PER_BATCH` parameter in both unfiltered and filtered mask data flows.